DataProducts/inc/CrvStatus.hh

```diff
@@ -0,0 +1,46 @@
+#ifndef RecoDataProducts_CrvStatus_hh
```

Comment:
Will this data product be created from the online data as well?  Or is it a summary that's created when the online data are reformatted as CRVDigi?